hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/AclEntryStatusFormat.java

```diff
@@ -47,7 +47,7 @@ public enum AclEntryStatusFormat implements LongBitFormat.Enum {
   private static final AclEntryType[] ACL_ENTRY_TYPE_VALUES =
       AclEntryType.values();
 
-  private final LongBitFormat BITS;
```

Comment:
Why remove  the “private”

Reply:
Because the “private” BITS cannot be import by FSImageFormatPBINode. This change may not be appropriate, I will write a getter for BITS is a better solution.